### PR TITLE
Chapter1/Homework 5

### PR DESCRIPTION
**Pull request by Tailor-AI**
First draft for homework 5.

### CODE REVIEW

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 1]
```diff
@@ -29,8 +61,13 @@ public static void Reverse(int[] array)
         /// <returns>A new array with the last element removed. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveLast(int[] array)
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs:62]: Reuse RemoveAt

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 2]
```diff
@@ -39,8 +76,15 @@ public static int[] RemoveLast(int[] array)
         /// <returns>A new array with the first element removed. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveFirst(int[] array)
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs:72]: Reuse RemoveAt

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 3]
```diff
@@ -63,8 +117,14 @@ public static int[] RemoveAt(int[] array, int index)
         /// <returns>A new array with element added at a given index. If an array is empty or null, returns new array with number in it.</returns>
         public static int[] InsertFirst(int[] array, int number)
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs:105]: Reuse InsertFirst

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 4]
```diff
@@ -75,8 +135,14 @@ public static int[] InsertFirst(int[] array, int number)
         /// <returns>A new array with element added in the end of array. If an array is empty or null, returns new array with number in it.</returns>
         public static int[] InsertLast(int[] array, int number)
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs:116]: Reuse InsertFirst

**Review by Almantask — CHANGES_REQUESTED** [review]
Reuse code more 🛩️

**Comment by Tailor-AI** [comment]
Reused RemoveAt and InsertAt in this second draft.

File: Src/BootCamp.Chapter1/ArrayOperations.cs (hunk 1 above)
Comment (Tailor-AI) [reply Src/BootCamp.Chapter1/ArrayOperations.cs:62]: ok, fixed

**Review by Almantask — APPROVED** [review]
Yes, that's exactly what I was expecting 😄

**Pull request closed without merging**